cub/device/dispatch/dispatch_radix_sort.cuh

```diff
@@ -1637,6 +1639,42 @@ struct DispatchRadixSort :
         return InvokeOnesweep<ActivePolicyT>();
     }
 
+    CUB_RUNTIME_FUNCTION __forceinline__
+    cudaError_t InvokeCopy()
+    {
+        // is_overwrite_okay == false here
+        // Return the number of temporary bytes if requested
+        if (d_temp_storage == nullptr)
+        {
+            temp_storage_bytes = 1;
+            return cudaSuccess;
+        }
+        
+        // Copy keys
+        cudaError_t error = cudaSuccess;
+        error = cudaMemcpyAsync(d_keys.Alternate(), d_keys.Current(), num_items * sizeof(KeyT),
```

Comment:
We have to synchronize in the case of `debug_synchronous`
```
if (debug_synchronous)
{
  if (CubDebug(error = SyncStream(stream))
  {
    // ..
```

Reply:
Done (using `DebugSyncStream()`).